sdk/azidentity/platform-matrix.json

```diff
@@ -0,0 +1,16 @@
+{
+    "include": [
+        {
+            "Agent": {
+                "msi_image": {
+                    "OSVmImage": "ubuntu-22.04",
+                    "Pool": "azsdk-pool-mms-ubuntu-2204-identitymsi"
+                }
+            },
+            "GoVersion": [
+                "1.18"
```

Comment:
Should this be expanded to include more versions?

Reply:
We should run on n and n-1 at a minimum.

Reply:
I think one suffices because we test this code with other versions offline and I can't imagine a difference between two versions that only a live test would reveal 🤔

Reply:
I tend to agree. Perhaps this should be the latest version though? We've already got 1.18 compat coverage elsewhere, so if we're going for single-version coverage it might be better to use the N version.